ui/apps/platform/src/Containers/NetworkGraph/api/useFetchNetworkBaselines.ts

```diff
@@ -23,9 +26,9 @@ const defaultResultState = {
  * of the supplied peers
  */
 function useFetchNetworkBaselines(deploymentId): FetchNetworkBaselinesResult {
-    const [result, setResult] = useState<FetchNetworkBaselinesResult>(defaultResultState);
+    const [result, setResult] = useState<Result>(defaultResultState);
 
-    useEffect(() => {
+    function fetchBaselines() {
         const networkBaselinesPromise = fetchNetworkBaselines({ deploymentId });
 
         networkBaselinesPromise
```

Comment:
Not part of this PR's changes, but could this be simplified by eliminating the "promise" var?

```suggestion
        fetchNetworkBaselines({ deploymentId })
            .then(...
```

Reply:
Yea we can do that